### PR TITLE
Create quarto book

### PR DESCRIPTION
**Pull request by rogin123**

### CODE REVIEW

File: intro_series_book/02_lesson.qmd [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+-   Assignment Operator
+-   Console/Environment/Script
+-   Comment your code and read your error messages
+
+# The tidyverse package
+
+-   Collection of packages using same syntax and grammar
+-   Works with "tidy" data
+-   Uses tibbles (https://r4ds.had.co.nz/tibbles.html)
```
Comment (awunderground) [inline intro_series_book/02_lesson.qmd:34]: Maybe turn this into a hyperlink?

File: intro_series_book/02_lesson.qmd [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+-   `summarize()`
+
+All of these functions have the same structure.
+
+1.  The first argument is the dataframe
+2.  The subsequent arguments describe what to do with the data frame, using the variable names (without quotes because you're already refering to that df)
+3.  Separate functions by %\>%
+4.  The result is a new data frame.
+
+![](C:/Users/fwalker/Downloads/image.png)
```
Comment (awunderground) [inline intro_series_book/02_lesson.qmd:54]: This file path doesn't work!

**Review by awunderground — CHANGES_REQUESTED** [review]
I reviewed the general setup and the earlier content. We can improve from here. A few macro comments:

1. Remove all `h1` markdown from `.qmd` files. The current section numbering doesn't work. 
2. You may want to change the YAML headers. It's based on R Markdown right now. In particular, I would switch `self-contained: TRUE` to `embed-resources: true`. 
3. The content is difficult to read with its current spacing. Try adding some css [from here](https://github.com/UI-Research/2023_jsm-data-privacy-training/blob/main/www/styles.css).

File: intro_series_book/02_lesson.qmd (hunk 1 above)
Comment (rogin123) [reply intro_series_book/02_lesson.qmd:34]: embedded

File: intro_series_book/02_lesson.qmd (hunk 2 above)
Comment (rogin123) [reply intro_series_book/02_lesson.qmd:54]: have no idea what the picture was supposed to be, and there wasn't a picture in the R4DS section linked so just deleted it

**Review by awunderground — APPROVED** [review]

**Pull request merged**